app/views/users/new.html.erb

```diff
@@ -22,7 +22,17 @@
         </div>
         <div class="card-body background">
           <% unless configured_providers.length.zero? %>
-            <% configured_providers.each do |provider| %>
+
+            <% if provider = configured_providers[0] %>
```

Comment:
A couple of details here. in the if condition, it should be == oinstead of =. As it is right now, both are always true.
<% if provider == configured_providers[0] %>

Second, I am not sure we want to do it this way, If the code has to be specific for google, we can make the comparision looking specifically for google as the tool provider (I don;t remember how the strategies are identified, but we'll figure it out)